version/v1.go

```diff
@@ -8,17 +8,13 @@ import (
 // Version1Routes registers all routes for the v1 version
 func Version1Routes(r *mux.Router) {
 
-	// this doesn't need versioning, yet
-	base.RegisterHealthRoutes(r)
+	//server info in json format
+	base.RegisterHomeRoute(r)
```

Comment:
Now, the home route doesn't have any version which is correct but why is it that we have the route functions inside the `v1` folder in `route/v1/base/home`?

Isn't it supposed to be `route/home.go`?

Reply:
I was thinking, since home is inside the base folder in the controller, Can I make the home route dir to be route/base/home.go instead of route/home.go?

Reply:
That means we'll have a base folder inside route and another inside v1 which is still inside route. Sounds odd 🤔

Reply:
Thanks for the clarity
I understand why I can't do it that way now. The branch has been updated.